src/adUnits.js

```diff
@@ -0,0 +1,18 @@
+import { deepAccess } from './utils';
+
+let adUnits = {};
+
+function incrementCounter(adunit) {
+  adUnits[adunit] = adUnits[adunit] || {};
+  adUnits[adunit].counter = (deepAccess(adUnits, `${adunit}.counter`) + 1) || 1;
+  return adUnits[adunit].counter;
```

Comment:
why return the count?

Reply:
It is increment and get rather than just increment. Will be useful for developer.